string command
float64 target_x
float64 target_y
float64 target_psi
bool station_keeping
float64 duration
---
string feedback
